x-pack/auditbeat/module/system/process/_meta/data.json

```diff
@@ -1,29 +1,42 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "agent": {
+    "beat": {
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
     "event": {
         "action": "existing_process",
         "dataset": "process",
-        "id": "5795d53b-f7c2-463c-9c04-f316ae876d51",
-        "module": "system",
-        "kind": "state"
+        "id": "4c8f57c4-d6d5-4b6d-9119-d47b68e474d6",
+        "kind": "state",
+        "module": "system"
     },
-    "message": "Process zsh (PID: 2363) is RUNNING",
+    "message": "Process zsh (PID: 9086) by user elastic is RUNNING",
     "process": {
         "args": [
-            "/usr/bin/zsh"
+            "zsh"
         ],
         "executable": "/bin/zsh",
         "name": "zsh",
-        "pid": 2363,
-        "ppid": 2362,
-        "start": "2018-12-10T16:36:25.21Z",
+        "pid": 9086,
+        "ppid": 9085,
+        "start": "2019-01-09T12:05:44.87Z",
         "working_directory": "/home/elastic"
     },
-    "service": {
-        "type": "system"
+    "system": {
+        "audit": {
+            "process": {
+                "egid": "1002",
+                "euid": "1002",
+                "gid": "1002",
+                "sgid": "1002",
+                "suid": "1002",
+                "uid": "1002"
+            }
+        }
+    },
+    "user": {
+        "id": "1002",
+        "name": "elastic"
```

Comment:
Note that ECS now defines the `group` field set as well. So you should also copy `system.audit.process.gid` to `group.id`.

Other than that, this event looks good

Reply:
Great, I've added `group.id` and `group.name`. I'm less comfortable having two duplicated fields (probably shouldn't even had one in the first place since we don't have those in other places either), so I've removed `system.audit.process.uid` and `system.audit.process.gid`.